src/commonMain/kotlin/exchange.dydx.abacus/functional/vault/VaultAccount.kt

```diff
@@ -77,14 +85,16 @@ object VaultAccountCalculator {
 
     fun calculateUserVaultInfo(
         vaultInfo: AccountVaultResponse,
-        vaultTransfers: IndexerTransferBetweenResponse
+        vaultTransfers: IndexerTransferBetweenResponse,
     ): VaultAccount {
         val presentValue = vaultInfo.equity?.let { it / 1_000_000 }
         val netTransfers = parser.asDouble(vaultTransfers.totalNetTransfers)
         val withdrawable = vaultInfo.withdrawableEquity?.let { it / 1_000_000 }
         val allTimeReturn =
             if (presentValue != null && netTransfers != null) (presentValue - netTransfers) else null
 
+        val impliedShareValue: Double = if (vaultInfo.shares?.numShares != null && presentValue != null) presentValue / vaultInfo.shares.numShares else 0.0
```

Comment:
can we also check if `vaultInfo.shares?.numShares` is not zero to prevent div by zero?